centreon/src/Core/Media/Infrastructure/Repository/DbReadMediaRepository.php

```diff
@@ -84,6 +86,47 @@ public function findById(int $mediaId): ?Media
         return $this->createMedia($record);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function findByIds(array $mediaIds): array
+    {
+        [$bindValues, $bindQuery] = $this->createMultipleBindQuery(
+            $mediaIds,
+            ':mediaId_',
+        );
```

Comment:
```suggestion
        if ($mediaIds === []) {
            return [];
        }

        [$bindValues, $bindQuery] = $this->createMultipleBindQuery(
            $mediaIds,
            ':mediaId_',
        );
```
OR there is be a risk to have a request IN () that will result in syntax error